base/docker/extension/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "ballerina"
+name = "extension"
+version = "0.1.0"
+distribution = "2201.9.2"
+
+[build-options]
+observabilityIncluded = true
```

Comment:
Adding a single file would be enough. We don't need the Ballerina.toml and the .gitignore files

Reply:
Updated https://github.com/ballerina-platform/module-ballerina-docker/pull/1051/commits/fef253c19d05fdfd5a58fa591de1b2c46d75f67e